test/unittest/blas1/blas1_sdsdot_test.cpp

```diff
@@ -35,7 +35,7 @@ template <typename scalar_t, helper::AllocType mem_alloc>
 void run_test(const combination_t<scalar_t> combi) {
   std::string alloc;
   index_t N;
-  float sb;
+  scalar_t sb;
```

Comment:
I think this still needs to be `float` instead of `scalar_t` since this argument is of type `float` in oneapi specification for `sdsdot`.

Reply:
Indeed! Will fix that.